LibuvSharp/UVStream.cs

```diff
@@ -211,7 +212,58 @@ public void Write(ArraySegment<byte> data, Action<Exception> callback)
 			Ensure.Success(r);
 		}
 
-		public void Shutdown(Action<Exception> callback)
+        public void Write(IList<ArraySegment<byte>> buffers, Action<Exception> callback)
```

Comment:
Why did you use an IList here? I'd rather use IEnumerable<T> because it is much more generic.